packages/core/src/spawn.ts

```diff
@@ -7,8 +7,8 @@ import {
   AnyStateMachine,
   Spawner
 } from '.';
-import { ObservableActorRef } from './ObservableActorRef';
-import { isString } from './utils';
+import { interpret } from './interpreter';
+import { isBehavior, isString } from './utils';
```

Comment:
this suggested change depends on [this one](https://github.com/statelyai/xstate/pull/3455/files#r1053712186)
```suggestion
import { isString } from './utils';
```